Assets/HoloToolkit/SpectatorView/Scripts/Utils/TweenAlpha.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.ARCapture
+{
+    /// <summary>
+    /// Utility for fading out marker
+    /// </summary>
+    public class TweenAlpha : MonoBehaviour
+    {
+        /// <summary>
+        /// Alpha value to fade to
+        /// </summary>
+        [Tooltip("Alpha value to fade to")]
+        [SerializeField]
+        private float targetAlpha;
+        /// <summary>
+        /// Alpha value to fade to
+        /// </summary>
+        public float TargetAlpha
+        {
+            get
+            {
+                return targetAlpha;
+            }
+
+            set
+            {
+                targetAlpha = value;
+            }
+        }
+
+        /// <summary>
+        /// Time taken to fade from current alpha to TargetAlpha
+        /// </summary>
+        [Tooltip("Time taken to fade from current alpha to TargetAlpha")]
+        [SerializeField]
+        private float duration = 0.5f;
+        /// <summary>
+        /// Time taken to fade from current alpha to TargetAlpha
+        /// </summary>
+        public float Duration
+        {
+            get
+            {
+                return duration;
+            }
+
+            set
+            {
+                duration = value;
+            }
+        }
+
+        /// <summary>
+        /// Material to operate on
+        /// </summary>
+        private Material mat;
+
+
+        void Start()
+        {
+            if (mat == null)
+            {
+                mat = GetComponent<Renderer>().material;
```

Comment:
Please destroy the material in `OnDestroy`.  This is a known memory leak issue in Unity